### DISCUSSION TITLE
How can I render a figure as a table (with caption)?

### DISCUSSION THREAD

**Discussion by sbacelar** [quarto-dev/quarto-cli#8759] [Q&A] [answered]
### Description

The second part of this code does not work.

````qmd
---
title: "figure as table"
format: html
editor: source
---

# Elephants


![Surus](elephants/surus.png)


```{r}
#| label: tbl-suruslabel
#| tbl-cap: "Surus"
"elephants/surus.png"
```
````

**Comment by mcanouil** [comment, marked as answer]
A string in R is just a string.
There is no way Quarto will read it as an image.
If you want to do that, then you have to use things such as `knitr::include_graphics` or output markdown yourself.

Also see <https://quarto.org/docs/blog/posts/2024-01-24-1.4-release/#cross-references>.

**Reply by sbacelar** [reply]
Yes, that was exactly the reference I was looking for. I remembered the table figure but not where it was. Thank you,

**Reply by mcanouil** [reply]
There is a search engine in the website.
There are complete guides to cross-reference and custom cross-reference objects.

**Reply by sbacelar** [reply]
I Know. I tried the search engine but could not find the "table figure".
